test/sandbox/sandbox.js

```diff
@@ -11,6 +12,12 @@ var compiledContainer = document.querySelector('#output .compiled');
 var outputContainer = document.querySelector('#output .output');
 var stringContainer = document.querySelector('#output .string');
 
+// Customize compiler
+
+compiler.useExtension(tdDebugger);
+
+compiler.ready();
```

Comment:
Can we change this API so that useExtension isn't even a thing.  This way when you call compiler.useExtensions, it will call ready on it's own right after.